modules/tensor_mechanics/tests/generalized_plane_strain/tests

```diff
@@ -1,10 +1,26 @@
 [Tests]
```

Comment:
Since you have modified material class of small total strain, incremental strain and finite strain for Generalized Plane Strain case, I would suggest you add several tests for those formulations.